README.md

```diff
@@ -193,50 +193,45 @@ Configuration is handled differently depending on how the connector is installed
+| client.proxy                      | Sets whether the configured proxy options are used.                                                                                                                                                                             | *Boolean* ||
+| client.proxy.host                 | The proxy host to use. Requires `client.proxy` to be set to `true`.                                                                                                                                                             | *String*  ||
+| client.proxy.non.proxy.hosts      | A list of hosts excluded from using the proxy. Requires `client.proxy` to be set to `true`.                                                                                                                                     | *List*    ||
+| client.proxy.username             | The client proxy username. Requires `client.proxy` to be set to `true`.                                                                                                                                                         | *String*  ||
+| client.proxy.password             | The client proxy password. Requires `client.proxy` to be set to `true`.                                                                                                                                                         | *String*  ||
+| client.proxy.port                 | The client proxy port. Requires `client.proxy` to be set to `true`.                                                                                                                                                             | *Integer* ||
+| client.proxy.pref.auth.type       | The authentication type to use with the client proxy. Must be one of `BASIC`, `DIGEST` or `X_ABLY_TOKEN`. Requires `client.proxy` to be set to `true`.                                                                          | *String*  | Basic   |
+| client.push.full.wait             | Sets whether Ably should wait for all the effects of push REST requests before responding.                                                                                                                                      | *Boolean* ||
+| client.tls                        | Sets whether TLS is used for all connection types.                                                                                                                                                                              | *Boolean* | True    |
+| client.token.params               | Sets whether the configured token parameters are used.                                                                                                                                                                          | *Boolean* ||
```

Comment:
Doesn't the connector always use key-based auth?

Reply:
Good question... In another branch, I've fixed our usage of `AblyRest` to pass in the whole `ClientOptions` object instead of just the key, as it was doing previously. I assume that means you could in theory provide a url for token auth? Only think is I think we currently mandate a value for `client.key` in our config validation. We don't really expose a way for the connector to identify itself to any other auth service though, AFAIK.

Perhaps we should just delete this config to avoid confusion?

Reply:
Yeh I think delete the token related config to avoid confusion